### ISSUE TITLE
Too much logging information by default

### ISSUE THREAD

**Issue by flemmel** [jkeger/arctic#40]
Dear all,

I just found a 'nasty' bug.

When 'arctic' is used (for example) to remove CTI, lots of debug information is shown even when the verbosity level is set to 0 .

So when we run our Python script using Arctic from the command line, we get a lot of logging information (it's fine) and the script works (that's good). But when we run the script using JupyterLab, the code is blocking (not good !) because there is too many logging information (even if it's not displayed in JupyterLab)

The only solution I found is to replace in the C++ code the following statements `print_v(0, "...");` with `print_v(1, "...")`.

I will create a new pull request for a possible solution.

Cheers,
Frederic

**Comment by rjmassey** [comment]
Sorry, that’s my bad. I accidentally pushed some debugging to the wrong branch. Will fix later.


On 16 Feb 2023, at 13:34, flemmel ***@***.***> wrote:

﻿
[EXTERNAL EMAIL]

Dear all,

I just found a 'nasty' bug.

When 'arctic' is used (for example) to remove CTI, lots of debug information is shown even when the verbosity level is set to 0 .

So when we run our Python script using Arctic from the command line, we get a lot of logging information (it's fine) and the script works (that's good). But when we run the script using JupyterLab, the code is blocking (not good !) because there is too many logging information (even if it's not displayed in JupyterLab)

The only solution I found is to replace in the C++ code the following statements print_v(0, "..."); with print_v(1, "...").

I will create a new pull request for a possible solution.

Cheers,
Frederic
